activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -87,9 +87,21 @@ def self.init_fallbacks(fallbacks)
         when Hash, Array
           Array.wrap(fallbacks)
         else # TrueClass
-          []
+          [I18n.default_locale]
         end
 
+      if args.empty? || args.first.is_a?(Hash)
+        ActiveSupport::Deprecation.warn(<<-MSG.squish)
+          Using I18n fallbacks with an empty `defaults` sets the defaults to
+          include the `defalt_locale`. This behaviour will change in Rails 6.1.
```

Comment:
Can you change `behaviour` to `behavior` per our [documentation requirements](https://guides.rubyonrails.org/api_documentation_guidelines.html#english).

Reply:
small typo in the `defalt_locale` -> `default_locale`

Reply:
@lsylvester I think you forgot to push.

Reply:
@rafaelfranca Thanks for letting me know. Sorry about that. Should be fixed now.